sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/MaxByAndMinBy.scala

```diff
@@ -100,6 +100,9 @@ abstract class MaxMinBy extends DeclarativeAggregate {
   """,
   since = "3.0.0")
 case class MaxBy(valueExpr: Expression, orderingExpr: Expression) extends MaxMinBy {
+
+  override def prettyName: String = "max_by"
+
   override protected def funcName: String = "max_by"
```

Comment:
Seems like you can remove this and replace it to `prettyName`.

Reply:
Yes. I will replace it.